spec/unit/spotlight/dor/indexer_spec.rb

```diff
@@ -214,7 +214,9 @@
       </mods>
     EOF
   end
-  describe "#add_folder_name" do
+
+  # rubocop:disable Metrics/LineLength
```

Comment:
`Metrics/LineLength` is about the number of characters in the line (and I've confused that with the other `Metrics/*Length` a number of times).

I looked at this particular case and thought that 169-character lines seemed pretty long, but it would be more inconsistent and/or harder to read if we split the hash across lines.